browser/src/VariantList/ExportVariantsButton.tsx

```diff
@@ -1,10 +1,79 @@
 import React from 'react'
 
 import { Button } from '@gnomad/ui'
+import { Population } from '../VariantPage/VariantPage'
 
-import { GNOMAD_POPULATION_NAMES, PopulationId } from '@gnomad/dataset-metadata/gnomadPopulations'
+import {
+  GNOMAD_POPULATION_NAMES,
+  PopulationId,
+  populationsInDataset,
+} from '@gnomad/dataset-metadata/gnomadPopulations'
+import { DatasetId, isExac, isV2, isV3, isV4 } from '@gnomad/dataset-metadata/metadata'
 
-const exportVariantsToCsv = (variants: Variant[], datasetId: any, baseFileName: any) => {
+type Column = {
+  label: string
+  getValue: (variant: VariantTableVariant) => string
+}
+
+export const createPopulationColumns = (datasetId: DatasetId) => {
+  /* eslint-disable no-nested-ternary */
+  const topLevelDataset = isV4(datasetId)
+    ? 'v4'
+    : isV3(datasetId)
+    ? 'v3'
+    : isV2(datasetId)
+    ? 'v2'
+    : isExac(datasetId)
+    ? 'ExAC'
+    : 'default'
+  /* eslint-enable no-nested-ternary */
+
+  const datasetPopulations: PopulationId[] = populationsInDataset[topLevelDataset]
+
+  let populationColumns: Column[] = []
+  datasetPopulations.forEach((popId: PopulationId) => {
+    const popName = GNOMAD_POPULATION_NAMES[popId]
+
+    populationColumns = populationColumns.concat([
+      {
+        label: `Allele Count ${popName}`,
+        getValue: (variant: VariantTableVariant) =>
+          variant.populations.filter((v) => v.id === popId)[0]
```

Comment:
Looks like the same logic apart from column name in a number of places, let's take this opportunity to extract a helper function to make that clearer.